_guides/quickstart-app-development.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Quickstart for DHIS2 App Development
+category: guide
+layout: page
+---
+
+## Create a new app and connect it to a local DHIS2 instance in 10 minutes or less.
```

Comment:
```suggestion
Create a new app and connect it to a local DHIS2 instance in 10 minutes or less.
```

Reply:
@deboragaleano just noticed that subtitles are very big on the current rendering of the page. I suggest removing the heading formatting here.

Reply:
Thanks @veganpolice, somehow I can't commit the suggested change but I can definitely add it when I update other pages 👍